docs/Functional.md

```diff
@@ -1,51 +1,121 @@
-<!--Objectives are written by product designers in response to the requirements. They describe in a more specific way what the product will look like. Objectives may describe architectures, protocols and standards to which the product will conform. Measurable objectives are those that set some criteria by which the end product can be judged. Measurability can be in terms of some index of customer satisfaction or in terms of capabilities and task times. Objectives must recognize time and resource constraints. The development schedule is often part or a corollary of the objectives.-->
 
 It will be a single web page with an input field. The user will be able to enter a postal address or a coordinate and the page will show the time for Santa to arrive on the page.
 The service will be able to scale depending on the number of requests per minute.
 It will be supported by the majority of current browsers OSs and devices. 
 
 ## User documentation.
-<!--In general, all of the preceding documents (except the logic specification) are used as source material for the technical manuals and online information, such as help pages, that are prepared for the product's users.-->
 
 Write your adress in the field. And press enter.
 
 ## Test plan.
-<!--Most development groups have a formal test plan that describes test cases that will exercise the programming that is written. Testing is done at the module (or unit) level, at the component level and at the system level in context with other products. This can be thought of as alpha testing. The plan may also allow for beta test. Some companies provide an early version of the product to a selected group of customers for testing in a "real world" situation.-->
 
 The test plan is written [here](./Quality_Assurance\TestPlan.md).
 
+## Marketing plan.
+
+**HAVE TO BE VALIDATED WITH JEAN-PHILIPPE AND FRANCK**
+
+### Graphical identity
+
+**TO BE DEFINE WITH JEAN-PHILIPPE**
+
+### Social media
+
+- [ ] Post on Linkedin
+- [ ] Post on Instagram by ALGOSUP
+- [ ] Post on Instagram by ALGOSC
+- [ ] Repost on Linkedin by Franck Jeannin
+- [ ] Repost on Linkedin by Eric L'Archevèque
+- [ ] Repost on Linkedin by ALGOSUP
+- [ ] Link in AlgoBot description
+
+### Physical exposure
+
+- [ ] Poster in the school
+- [ ] QR Code at Epipapu (Vierzon)
+
+### Event exposure
+
+- [ ] Website displayed on the live of ALGOSC for the Telethon
+
+
+### Will not be done
+
+We don't want to limit our users base to be in Vierzon. So we will not do any physical exposure in Vierzon.
+We prefer to keep it online and accessible to everyone.
+
 ## The final product.
-<!--Ideally, the final product is a complete implementation of the functional specification and design change requests, some of which may result from formal testing and beta testing.-->
 
-The final product can be found at [xmas.algosup.com](https://xmas.algosup.com/).
\ No newline at end of file
+The final product can be found at [xmas.algosup.com](https://xmas.algosup.com/).
+
```

Comment:
What the final product will looks like ? ( Mock-up )

If someone is blind, is there something for that ?

Reply:
"If someone is blind, is there something for that ?"
stop trolling please

Reply:
For people Visually handicapped for example